code/nrf-connect/prstlib/src/shtc3.c

```diff
@@ -0,0 +1,53 @@
+#include "prstlib/shtc3.h"
+
+#include <zephyr/drivers/i2c.h>
+#include <zephyr/logging/log.h>
+#include <zephyr/zephyr.h>
```

Comment:
<zephyr/kernel.h> instead, sorry for nagging but I really like a warning-less build :)

Reply:
Seems like my grep-foo failed me. Done, tks! I really appreciate you helping out here.